diff --git a/node_modules/@silveredgold/beta-shared-components/lib/beta-shared-components.es.js b/node_modules/@silveredgold/beta-shared-components/lib/beta-shared-components.es.js
index f60eb2a..d741d42 100644
--- a/node_modules/@silveredgold/beta-shared-components/lib/beta-shared-components.es.js
+++ b/node_modules/@silveredgold/beta-shared-components/lib/beta-shared-components.es.js
@@ -22,7 +22,6 @@ var __publicField = (obj, key, value) => {
   return value;
 };
 import { inject, defineComponent, toRefs, watch, openBlock, createBlock, unref, withCtx, createVNode, createElementBlock, renderSlot, createCommentVNode, Fragment, renderList, createTextVNode, ref, computed, onBeforeMount, createElementVNode, toDisplayString, reactive, h } from "vue";
-import { notificationApiInjectionKey } from "naive-ui/lib/notification/src/NotificationProvider";
 import { useNotification, NCard, NTabs, NTabPane, NForm, NFormItem, NInputGroup, NSelect, NSlider, NPopover, NIcon, NThing, NSpace, NCheckbox, NResult, NButton, NRadioGroup, NRadioButton, NText, NGrid, NGridItem, NAlert, NDataTable, NPageHeader, NAvatar, NGi, NStatistic, NTooltip, NTree, NIconWrapper, NImage, useThemeVars, NSpin } from "naive-ui";
 import { AlertCircleOutline, HelpCircleOutline, InformationCircle, Open, Settings, Refresh, Help, Checkmark, Close, Images, Stop } from "@vicons/ionicons5";
 import { getCensorTypes } from "@silveredgold/beta-shared/preferences";
@@ -2012,8 +2011,9 @@ const _sfc_main = /* @__PURE__ */ defineComponent({
   }
 });
 function useOptionalNotification() {
-  const notif = inject(notificationApiInjectionKey, void 0);
-  return notif;
+  const notif = useNotification();
+  // const notif = inject(notificationApiInjectionKey, void 0);
+  return notif || undefined;
 }
 const censorBackend = Symbol();
 function useBackendTransport() {
